lib/routes/people-routes.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router(); // app
+const logger = require('../../logger.js');
+
+const People = require('../models/people.js');
+let people = new People();
+
+// GET with Promises
+/**
+ * Route to request a specific person
+ * @group People 
+ * @route GET '/people'
+ * @security basicAuth
+ * @returns {object} 200 { count: 2, results: [ {}, {} ] }
```

Comment:
This doesn't seem quite accurate because you're not returning an object with a key `count`.

Your `getFromField` function should just return an array of records.